app/javascript/react/components/NetflixShowContainer.js

```diff
@@ -24,14 +26,75 @@ const NetflixShowContainer = (props) => {
       .catch(error => console.error(`Error in fetch: ${error.message}`))
   }, [])
 
+  const addNewReview = (props) => {
+    fetch(`/api/v1/netflix_shows/${id}/reviews`, {
```

Comment:
this path looks great! BUT, make sure to go back to week 6 curriculum and look at the article `Fetch in a React App within Rails`, because you will need to add a bit more to the rest of the fetch